### PR TITLE
docs #78: add cmake to the dependency

### PR DESCRIPTION
**Pull request by SteveLauC**

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -2,29 +2,36 @@
...
+    rustup default nightly
+    ```
+
+    or
+
+    ```
+    rustup toolchain install nightly
+    rustup component add rustfmt --toolchain nightly
+    ```
+* [Cmake](https://cmake.org/install/)
```
Comment (roseboy-liu) [inline docs/get-started.md:19]: more detail about ‘cmake install’

**Review by roseboy-liu — COMMENTED** [review]
Thank you for your pr

File: docs/get-started.md (hunk 1 above)
Comment (SteveLauC) [reply docs/get-started.md]: I just pushed another commit to add some details about the installation of cmake:)

**Review by roseboy-liu — APPROVED** [review]
LGTM

**Comment by roseboy-liu** [comment]
move to https://github.com/cnosdb/cnosdb

**Pull request merged**